### DISCUSSION TITLE
Datasets to integrate

### DISCUSSION THREAD

**Discussion by hariszaf** [msysbio/bacterial_growth#25] [Q&A]
Here, we keep track of already published, public datasets that we could/should add on the database. 
Please, provide a link if possible. 

We could start from the studies used in the FBA-based microbial interaction evaluation paper:

https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-024-05651-7/tables/2

@hallucigenia-sparsa 
@AndrewRadev

**Comment by hallucigenia-sparsa** [comment]
Yes, good idea, plus of course the in-house data sets:
- https://www.nature.com/articles/s41396-023-01501-1
- https://www.biorxiv.org/content/10.1101/2024.11.28.625814v1
- https://www.tandfonline.com/doi/full/10.1080/19490976.2022.2155019
- https://www.nature.com/articles/s43705-022-00123-6

Ideally a task for Sourav if he goes through with his internship
